transactions.js

```diff
@@ -293,7 +293,7 @@ const TransactionValues = {
     /**
      * Unspecified.
      */
-    UNSPECIFIED: 'UNSPECIFIED',
+    PAYMENT_TYPE_UNSPECIFIED: 'PAYMENT_TYPE_UNSPECIFIED',
```

Comment:
Thanks for the fix!

But because in the client library before it was using the key value `UNSPECIFIED`, renaming it to `PAYMENT_TYPE_UNSPECIFIED` would break people's code.

Can you also add another key named `UNSPECIFIED` that maps to `PAYMENT_TYPE_UNSPECIFIED` and leave a comment about it being there for backwards compatibility of a bug fix?